packages/react-native-client/src/hooks/usePeerStatus.ts

```diff
@@ -19,15 +19,10 @@ export type PeerStatus = 'connecting' | 'connected' | 'error' | 'idle';
  * @group Hooks
  */
 export const usePeerStatus = () => {
-  const [peerStatus, setPeerStatus] = useState(
+  const peerStatus = useFishjamEventState<PeerStatus>(
+    ReceivableEvents.PeerStatusChanged,
```

Comment:
**question:** It looks like PeerStatus was previously possible to be undefined. Can you confirm that it won't happen?

Reply:
This value on native side is never optional. The only possibility for it to be undefined is if `PeerStatus` on the native side would have different value. I don't know if it's possible for us to desynchronise native and RN